packages/controller-utils/src/create-service-policy.test.ts

```diff
@@ -0,0 +1,589 @@
+import { useFakeTimers } from 'sinon';
```

Comment:
Nit: can't we just use jest fake timers ?

Reply:
No, because we're still using an older version of Jest, and it doesn't support the "async" varieties of `next` and `tick`.